shared/constants/chat.js

```diff
@@ -144,10 +141,14 @@ export type InboxState = Record<{
 
 export type MetaData = Record<{
   fullname: string,
+  brokenTracker: boolean,
```

Comment:
Could the following status be merged into this record? Then you'd only have to thread one user info argument through the views instead of two.